src/wombats/sockets/game.clj

```diff
@@ -212,7 +211,9 @@
       (let [formatted-message {:username github-username
                                :message message
                                :color (get-player-color game-id chan-id)
-                               :timestamp (str (l/local-now))}]
+                               :timestamp (->> (t/now)
```

Comment:
Probably would be better to import `format-date` (and move it in a better spot), but looks good!